src/server/req_stat.c

```diff
@@ -638,77 +578,18 @@ req_stat_node(struct batch_request *preq)
 	preply->brp_choice = BATCH_REPLY_CHOICE_Status;
 	CLEAR_HEAD(preply->brp_un.brp_status);
 
-	memset(&attr, 0, sizeof(attr));
+	if (type == 0) {		/*get status of the named node*/
+		rc = status_node(pnode, preq, &preply->brp_un.brp_status);
 
-	if (preq->rq_extend != NULL) {
-		rc = sched_attr_def[SCHED_ATR_partition].at_decode(&attr, ATTR_partition, NULL, preq->rq_extend);
-		if (rc) {
-			(void) reply_free(preply);
-			req_reject(rc, bad, preq);
-			return;
-		}
-	}
+	} else {			/*get status of all nodes     */
```

Comment:
put a space after the /* and remove the extra spaces before the */

Reply:
Done